force-app/main/default/aura/contactForm/contactFormController.js

```diff
@@ -0,0 +1,16 @@
+({
+	clickCreate: function(component, event, helper) {
+        let validExpense = component.find('contactform').reduce(function (validSoFar, inputCmp) {
+            // Displays error messages for invalid fields
+            inputCmp.showHelpMessageIfInvalid();
+            return validSoFar && inputCmp.get('v.validity').valid;
+        }, true);
+        // If we pass error checking, do some real work
+        if(validExpense){
+            // Create the new expense
+            let newContact = component.get("v.newContact");
```

Comment:
Do we need to pass it from here to a helper or can get it directly in helper method